source/type/type.cpp

```diff
@@ -554,6 +569,12 @@ const VehicleJourney* VehicleJourney::get_corresponding_base() const {
     return nullptr;
 }
 
+std::vector<StopTime>::const_iterator VehicleJourney::earliest_stop_time() const {
```

Comment:
Can we have a vj with no stop_time? The clean method of data.cpp delete vj with empty stop_time_list, and realtime.cpp seems to not handle realtime message with empty aux_info.stop_times, so I guess not?